README.md

```diff
@@ -5,6 +5,7 @@ A generic DAO (Decentralized Autonomous Organization) written in Solidity to run
 Simple, decentralized and 100% secure.
 Feel free to reuse to create your own Decentralized organization.
 
+• **Scientific Reference:** *"Decentralized autonomous organization to manage a trust" -* [White Paper (Draft)](https://github.com/slockit/DAO/tree/master/paper)
```

Comment:
- No need for the opening bullet point.
- Please change `Scientific Reference` to `Reference`
- Maybe link to the white paper as a pdf? Since the link you are putting is to a directory containing only the source TeX files.